src/manage_acl.c

```diff
@@ -124,37 +125,44 @@ acl_user_can_super_everyone (const char *uuid)
   gchar *quoted_uuid;
 
   quoted_uuid = sql_quote (uuid);
-  if (sql_int (" SELECT EXISTS (SELECT * FROM permissions"
-               "                WHERE name = 'Super'"
-               /*                    Super on everyone. */
-               "                AND (resource = 0)"
-               "                AND subject_location"
-               "                    = " G_STRINGIFY (LOCATION_TABLE)
-               "                AND ((subject_type = 'user'"
-               "                      AND subject"
-               "                          = (SELECT id FROM users"
-               "                             WHERE users.uuid = '%s'))"
-               "                     OR (subject_type = 'group'"
-               "                         AND subject"
-               "                             IN (SELECT DISTINCT \"group\""
-               "                                 FROM group_users"
-               "                                 WHERE \"user\""
-               "                                       = (SELECT id"
-               "                                          FROM users"
-               "                                          WHERE users.uuid"
-               "                                                = '%s')))"
-               "                     OR (subject_type = 'role'"
-               "                         AND subject"
-               "                             IN (SELECT DISTINCT role"
-               "                                 FROM role_users"
-               "                                 WHERE \"user\""
-               "                                       = (SELECT id"
-               "                                          FROM users"
-               "                                          WHERE users.uuid"
-               "                                                = '%s')))));",
-               quoted_uuid,
-               quoted_uuid,
-               quoted_uuid))
+  if (
+    sql_int (
+      " SELECT EXISTS (SELECT * FROM permissions"
+      "                WHERE name = 'Super'"
+      /*                    Super on everyone. */
+      "                AND (resource = 0)"
+      "                AND subject_location"
+      "                    = " G_STRINGIFY (
+        LOCATION_TABLE) "                AND ((subject_type = 'user'"
```

Comment:
oh boy.  i think we need to use a %i for LOCATION_TABLE, to make this easier for clang.